gateway/src/main/scala/filodb/gateway/conversion/InputRecord.scala

```diff
@@ -96,17 +96,18 @@ object PrometheusInputRecord {
 
   // Create PrometheusInputRecords from a TimeSeries protobuf object
   def apply(tsProto: TimeSeries): Seq[PrometheusInputRecord] = {
-    val tags = (0 until tsProto.getLabelsCount).map { i =>
+    val tags = collection.mutable.HashMap[String, String]()
+    for ( i <- 0 until tsProto.getLabelsCount) {
```

Comment:
This change is not necessary - it creates a map from the tags in the photo, which is exactly what the old code did.  However, the old code is shorter and creates an immutable map, which is preferable.

Reply:
Please ignore the above change. It was part of an old commit where i was testing something.